TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+
+Change the foreground color of the profile. This overrides `foreground` set in the color scheme if `colorScheme` is set.
+
+**Property name:** `foreground`
+
+**Necessity:** Optional
+
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
+
+### Background color
+
+Change the background color of the profile with this setting. This overrides `background` set in the color scheme if `colorScheme` is set.
+
+**Property name:** `background`
+
+**Necessity:** Optional
+
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
+
+### Selection background color
+
+Set the background color of a selection within the profile. This will override the `selectionBackground` set in the color scheme if `colorScheme` is set.
+
+**Property name:** `selectionBackground`
+
+**Necessity:** Optional
+
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
+
+<br />
+
+___
+
+## Acrylic settings
+
+### Enable acrylic
+
+When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. The transparency only applies to focused windows due to OS limitation.
+
+**Property name:** `useAcrylic`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `false`
+
+### Acrylic opacity
+
+When `useAcrylic` is set to `true`, this sets the transparency of the window for the profile. Accepts floating point values from 0-1.
+
+**Property name:** `acrylicOpacity`
+
+**Necessity:** Optional
+
+**Accepts:** Number as a floating point value from 0-1
+
+**Default value:** `0.5`
+
+<br />
+
+___
+
+## Background image settings
+
+More detailed information can be found on the [background images page](./../background-images.md).
+
+### Setting the background image
+
+This sets the file location of the image to draw over the window background.
+
+**Property name:** `backgroundImage`
+
+**Necessity:** Optional
+
+**Accepts:** File location as a string
+
+### Background image stretch mode
+
+You can set how the background image is resized to fill the window.
+
+**Property name:** `backgroundImageStretchMode`
+
+**Necessity:** Optional
+
+**Accepts:** `"none"`, `"fill"`, `"uniform"`, `"uniformToFill"`
+
+**Default value:** `"uniformToFill"`
+
+### Background image alignment
+
+This sets how the background image aligns to the boundaries of the window.
+
+**Property name:** `backgroundImageAlignment`
+
+**Necessity:** Optional
+
+**Accepts:** `"center"`, `"left"`, `"top"`, `"right"`, `"bottom"`, `"topLeft"`, `"topRight"`, `"bottomLeft"`, `"bottomRight"`
+
+**Default value:** `"center"`
+
+### Background image opacity
+
+You can set the transparency of the background image.
+
+**Property name:** `backgroundImageOpacity`
+
+**Necessity:** Optional
+
+**Accepts:** Number as a floating point value from 0-1
+
+**Default value:** `1.0`
+
+<br />
+
+___
+
+## Scroll settings
+
+### Scrollbar visibility
+
+Set the visibility of the scrollbar.
```

Comment:
It's "always visible" and "hidden when not rolled-over", right? We should probably make that clear.

Reply:
Also needs the default specified.

Reply:
It’s actually “visible only when you hover over it” or “Literally never visible” because we couldn’t get “always visible all the time” working